tequilapi/http_api_server_test.go

```diff
@@ -0,0 +1,16 @@
+package tequilapi
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestLocalApiServerPortIsAsExpected(t *testing.T) {
+	server, err := NewServer("", 31337, nil)
+	assert.Nil(t, err)
+
+	assert.Equal(t, 31337, server.Port())
```

Comment:
You dont need a getter method, because You are testing in same package

Reply:
It's just considence. Port() method is part of interface and it was not intended just for testing.